src/ReactWrapper.jsx

```diff
@@ -722,11 +722,19 @@ class ReactWrapper {
    * @param {*} initialValue - the initial value
    * @returns {*}
    */
-  reduce(fn, initialValue) {
-    return this.getNodes().reduce(
-      (accum, n, i) => fn.call(this, accum, this.wrap(n), i),
-      initialValue,
-    );
+  reduce(fn) {
```

Comment:
I'll do one of the two - as long as the behavior and function length are correct, I'm happy.